share/we-lang/we-lang.go

```diff
@@ -1172,7 +1172,7 @@ func main() {
             space := strings.Repeat(" ", 93)
```

Comment:
RTL is already aligned:
![Screenshot from 2020-06-27 21-38-08](https://user-images.githubusercontent.com/697976/85930806-8ce05d80-b8bf-11ea-9358-14e59a699aa5.png)